suite-common/icons/assets/everstakeLogo.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="102" height="90" viewBox="0 0 102 90" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please make it be black by default. So that you can see the component preview in filesystem
![Screenshot 2025-06-17 at 13 07 49](https://github.com/user-attachments/assets/c4e43a9e-2f39-4fab-949a-03adad81b704)